scripts/package-cli

```diff
@@ -8,13 +8,32 @@ cd $(dirname $0)/..
 GO=${GO-go}
 
 for i in containerd crictl kubectl k3s-agent k3s-server k3s-token k3s-etcd-snapshot k3s-secrets-encrypt k3s-certificate k3s-completion; do
```

Comment:
containerd, crictl and kubectl do not work on Windows after .exe suffix is added. Function init() in /cmd/server/main.go needs to be updated to something like below

func init() {
	reexec.Register("containerd"+BINARY_POSTFIX, containerd.Main)
	reexec.Register("kubectl"+BINARY_POSTFIX, kubectl2.Main)
	reexec.Register("crictl"+BINARY_POSTFIX, crictl2.Main)
	reexec.Register("ctr"+BINARY_POSTFIX, ctr2.Main)
}